.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,68 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "blog", "core"]
+assignees:
+  - gt-codes
+  - darrylbrooks97
+body:
+  - type: checkboxes
+    id: Code-of-conduct
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/)
```

Comment:
Can you create a page on our site and add the link here? Ex: `blacc.xyz/conduct`

Reply:
https://core-atbcwuf7k-blacc.vercel.app/conduct